runtime/compiler/optimizer/JProfilingValue.cpp

```diff
@@ -343,12 +374,13 @@ TR_JProfilingValue::lowerCalls()
          // Extract the arguments and add the profiling trees
          TR::Node *value = child->getFirstChild();
          TR_AbstractHashTableProfilerInfo *table = (TR_AbstractHashTableProfilerInfo*) child->getSecondChild()->getAddress();
-         addProfilingTrees(comp(), cursor, value, table, NULL, true, trace());
-
+         addProfilingTrees1(comp(), cursor, value, table, child->getChild(2)->getConst<int32_t>() == NEED_NULLTEST, true, trace());
```

Comment:
Re: https://github.com/eclipse/openj9/pull/6585/files#r307494771 , Here is the place where I use `NEED_NULLTEST`